addons/ui/fnc_notify.sqf

```diff
@@ -0,0 +1,122 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_notify
+
+Description:
+    Displays a small notification to the player
+
+Parameters:
+    title of the notification, displayed in the top left
+    message displayed in the center
+    (optional) color of the notification title
+
+Returns:
+    Nothing
+
+Examples:
+    (begin example)
+        ["ARC", "Not Connected to Discord", [1,0,0,1]] call cba_fnc_notify;
+        ["CBA", "This text is white!"] call cba_fnc_notify;
+    (end)
+
+Author:
+    SynixeBrett
+
+---------------------------------------------------------------------------- */
+SCRIPT(notify);
+
+if !(hasInterface) exitWith {  };
+
+params [
+    ["_title", ""],
+    ["_message", ""],
+    ["_color", [1,1,1,1]]
+];
+
+if !(canSuspend) exitWith {[_title, _message, _color] spawn CBA_fnc_notify};
+
+disableSerialization;
+
+// Require a _title and _message
+
+if (_title isEqualTo "" && {_message isEqualTo ""}) exitWith {};
+
+private _display = findDisplay 46;
+
+private _index = uiNamespace getVariable [QGVAR(NotificationIndex), 0];
+uiNamespace setVariable [QGVAR(NotificationIndex), _index + 1];
+
+waitUntil {_index - GVAR(NotificationCurrentIndex) < GVAR(NotificationMax)};
+
+private _position = _index % GVAR(NotificationMax);
+private _safezoneX = safezoneW + safezoneX;
+
+// Create the 3 displays in the offscreen position
+
+private _ctrlBackground = _display ctrlCreate ["RscPicture", 20];
+_ctrlBackground ctrlSetPosition [
+    _safezoneX,
+    (0.24 + (_position * 0.08)) * safezoneH + (safezoneY * -0.5) ,
+    0.1 * safezoneW,
+    0.06 * safezoneH
+];
+_ctrlBackground ctrlSetText "#(argb,8,8,3)color(0,0,0,0.75)";
+_ctrlBackground ctrlCommit 0;
+
+private _ctrlTitle = _display ctrlCreate ["RscStructuredText", 10];
+_ctrlTitle ctrlSetPosition [
+    _safezoneX,
+    (0.24 + (_position * 0.08)) * safezoneH + (safezoneY * -0.5),
+    0.1 * safezoneW,
+    0.0255 * safezoneH
+];
+_ctrlTitle ctrlSetStructuredText parseText _title;
+_ctrlTitle ctrlSetTextColor _color;
+_ctrlTitle ctrlCommit 0;
+
+private _ctrlNotification = _display ctrlCreate ["RscStructuredText", 30];
+_ctrlNotification ctrlSetPosition
+[
```

Comment:
Inconsistent formatting. I prefer L57 & L67, and that is the style used in CBA in most places.